### DISCUSSION TITLE
Add Algolia Search to the site for searching static content inside the files

### DISCUSSION THREAD

**Discussion by ashishkharche** [spencerwooo/onedrive-vercel-index#847] [Ideas]
The current search feature only allows searching file names.

It does not support searching content inside the files or the page at all.

A feature request to implement Algolia Doc Search or Algolia Search please!

Thank you!

**Comment by spencerwooo** [comment]
Sorry, but that's not how search works here. Search relies on the upstream OneDrive API and what it is capable of. You cannot use Algolia on a serverless platform to perform search indexing in the background. So, bringing in Algolia is not feasible atm.
